widget/embedded/src/store/slices/wallets.ts

```diff
@@ -245,6 +250,76 @@ export const createWalletsSlice: StateCreator<
       });
     }
   },
+  fetchCustomTokensBalance: async (params) => {
+    const { tokens } = params;
```

Comment:
NIT: Destructuring assignments could make this code more readable.

```suggestion
    const { tokens, connectedWallets } = params;
```